plugins/outputs/influxdb/http.go

```diff
@@ -432,6 +433,7 @@ func makeQueryURL(loc *url.URL) string {
 		u.Host = "127.0.0.1"
 		u.Path = "/query"
 	case "http":
+	case "https":
 		u.Path = path.Join(u.Path, "query")
 	}
```

Comment:
I really hate that there is no default case that logs an error, like here:

https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb/influxdb.go#L163

Reply:
That's a good point, I added some handling of the default case: df78133bf3d20edaf20fb565559a0b4109b09779